alpha-build-machinery/make/targets/openshift/crd-schema-gen.mk

```diff
@@ -0,0 +1,36 @@
+CRD_SCHEMA_GEN_VERSION ?=v0.2.1
+CRD_SCHEMA_GEN_APIS ?=$(error CRD_SCHEMA_GEN_APIS is required) 
+CRD_SCHEMA_GEN_TEMP ?=_output/tools/src/sigs.k8s.io/controller-tools
+CRD_SCHEMA_GEN_OUTPUT ?=./manifests
+
+crd-schema-gen:
+	if [ ! -d $(CRD_SCHEMA_GEN_TEMP) ]; then \
+		mkdir -p $(CRD_SCHEMA_GEN_TEMP); \
+		git clone -b $(CRD_SCHEMA_GEN_VERSION) --single-branch --depth 1 https://github.com/kubernetes-sigs/controller-tools.git $(CRD_SCHEMA_GEN_TEMP); \
+		mkdir -p $(CRD_SCHEMA_GEN_TEMP)/bin; \
+		export GOPATH=$(shell pwd)/_output/tools/ && export GOBIN=$(shell pwd)/_output/tools/bin/ && cd $(CRD_SCHEMA_GEN_TEMP) && go mod vendor && go install -mod=vendor ./cmd/controller-gen/...; \
+	fi
+.PHONY: crd-schema-gen
+
+update-codegen-crds: crd-schema-gen
+	$(shell pwd)/_output/tools/bin/controller-gen \
+		schemapatch:manifests=./manifests \
+		output:dir=$(CRD_SCHEMA_GEN_OUTPUT) \
+		paths="$(subst $() $(),;,$(CRD_SCHEMA_GEN_APIS))"
+.PHONY: update-codegen-crds
+
+# crd-schema-patch should be used for repos with yaml patches. It calls update-codegen-crds then applies any patches after first copying any patch files to the output directory (to ensure that verify works since that target runs in a temp dir). It depends on the `yq` tool to merge yaml
+crd-schema-patch: update-codegen-crds
```

Comment:
rename it to `crd-schema-update-with-patch`?  so `verify-codegen-crds: crd-schema-patch` makes more sense when you read it?